.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,27 +10,27 @@ optimize: A new optimization
 fix: A bug fix
 perf: A code change that improves performance
 refactor: A code change that neither fixes a bug nor adds a feature
-style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+style: Changes that do not affect the meaning of the code (white space, formatting, missing semi-colons, etc)
 test: Adding missing tests or correcting existing tests
 chore: Changes to the build process or auxiliary tools and libraries such as documentation generation
 -->
 
 #### Check the PR title.
 <!--
-The description of title will be attached in Release Notes, 
+The description of the title will be attached in Release Notes, 
 so please describe it from user-oriented, what this PR does / why we need it.
-Please check your PR title with below requirements:
+Please check your PR title with the below requirements:
 -->
-- [ ] The PR title match the format: \<type\>(optional scope): \<description\>
-- [ ] The description of PR title is from user-oriented and clear enough for others to understand.
+- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
+- [ ] The description of this PR title is user-oriented and clear enough for others to understand.
 
 
 #### (Optional) Translate the PR title into Chinese.
 
 
 #### (Optional) More detail description for this PR(en: English/zh: Chinese).
 <!--
-Provide more detail info for review. If it is a perf type PR, perf data is suggested to give.
+Provide more detailed info for review. If it is a perf type PR, perf data is suggested to give.
```

Comment:
It's recommended to provide perf data if this is a perf type PR.

Reply:
done